src/main/java/de/retest/web/screenshot/ViewportOnlyMinimalScreenshot.java

```diff
@@ -0,0 +1,35 @@
+package de.retest.web.screenshot;
+
+import java.awt.image.BufferedImage;
+
+import org.openqa.selenium.WebDriver;
+
+import com.assertthat.selenium_shutterbug.core.Shutterbug;
+
+import de.retest.recheck.ui.image.ImageUtils;
+
+/**
+ * The default implementation that takes a screenshot of only the viewport visible to the user from the top page.
+ */
+public class ViewportOnlyMinimalScreenshot implements ScreenshotProvider {
+
+	static final int DEFAULT_WANTED_WIDTH_PX = 800;
+	private static final String RESIZE_MAX_WIDTH_PX = "de.retest.recheck.web.resizeMaxWidthPx";
```

Comment:
I assume, we keep this internal for now?
```suggestion
	private static final String RESIZE_MAX_WIDTH_PX = "de.retest.recheck.web.screenshot.maxWidthPx";
```

Reply:
Yeah, would do that.